src/pages/components/Overlay.qml

```diff
@@ -417,6 +417,11 @@ Background {
 
             onStatusChanged: browserPage.tabPageActive = (status == PageStatus.Active)
 
+            Component.onCompleted: {
```

Comment:
I don't understand the 'looks like obsolete' part. If I remove it here I have no changes. If I remove it from TabView I'm relying on undefined execution order being defined as I see it happening.